.github/workflows/merge-queue.yml

```
@@ -3,8 +3,92 @@
+        with:
+          node-version-file: 'package.json'
+
+      - name: Setup yarn
+        run: corepack enable
+
+      - name: Get cache
+        id: get-cache
+        uses: ./.github/actions/get-cache
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          keys: ${{ needs.prepare.outputs.CACHE_KEYS }}
```

actionlint (1.7.4)

property "cache_keys" is not defined in object type {test_chunks: string}

(expression)